cnames_active.js

```diff
@@ -3604,6 +3604,7 @@ var cnames_active = {
   "xterminal": "henryhale.github.io/xterminal",
   "xto6": "mohebifar.github.io/xto6", // noCF? (don´t add this in a new PR)
   "xtype": "lucono.github.io/xtypejs", // noCF? (don´t add this in a new PR)
+  "xvrtech": "https://fullstackedrvc.github.io/Agency/",
```

Comment:
```suggestion
  "xvrtech": "fullstackedrvc.github.io/Agency",
```
You do not need the `https://` nor the trailing slash